packages/components/src/components/dataDisplay/Tag/types.ts

```diff
@@ -1,10 +1,13 @@
 import { TagProps as ChakraTagProps, ResponsiveValue } from '@chakra-ui/react';
 
 export const tagSizes = ['s', 'm', 'l'] as const;
+export const tagVariants = ['subtle', 'outline', 'solid', 'custom'] as const;
 
 export type TagSize = (typeof tagSizes)[number];
+export type TagVariant = (typeof tagVariants)[number] | ChakraTagProps['variant'];
```

Comment:
why do we need the `ChakraTagProps` reference in here?

Reply:
not really needed